spec/robots/dor_repo/gis_delivery/load_geoserver_spec.rb

```diff
@@ -856,6 +856,37 @@
           expect(stubbed_layer_put).to have_been_requested
         end
       end
+
+      context 'when raster greyscale' do
+        let(:coverage) { instance_double(Geoserver::Publish::Coverage) }
+        let(:coverage_store) { instance_double(Geoserver::Publish::CoverageStore) }
+        let(:style) { instance_double(Geoserver::Publish::Style) }
+        let(:layer) { instance_double(Geoserver::Publish::Layer) }
+
+        before do
+          allow(GisRobotSuite).to receive(:determine_raster_style).and_return('grayscale_8_4')
+          allow(Geoserver::Publish::Coverage).to receive(:new).and_return(coverage)
```

Comment:
We do not have a TIF in the fixtures that has this raster_style computed from it.  We will just mock this result and all of the other outbound calls.  `GisRobotSuite` has test coverage for the `determine_raster_style` method.

Reply:
@kimdurante is it feasible to get TIFF files for this code path?  I would prefer not to have to mock everything out, but if we can't easily get reasonably sized TIFFs (< 100MB) then it'll have to do .